src/psij/testing/service.py

```diff
@@ -382,6 +382,75 @@ def run(self, site_id, run_id) -> object:
         add_cors_headers()
         return resp
 
+    @cherrypy.expose
+    @cherrypy.tools.json_out()
+    def getLatestResultForTestOnAllSites(self, sitesToGet) -> object:
+
+        import json
+        sitesToGet = json.loads(sitesToGet)
+
+        # Example sitesToGet = ["mothra.hidden.uoregon.edu", "reptar.hidden.uoregon.edu", "saturn.hidden.uoregon.edu"]
+        resp = {}
+
+        for site_id in sitesToGet:
+            resp[site_id] = self.getSite(site_id)
+
+        return resp
+
+
+    def getSite(self, site_id):
+        #  Example site_id="mothra.hidden.uoregon.edu"
+        run_id=""
```

Comment:
We seem to have no linting checks enabled on this repo, but I would suggest to add spaces around the assignment operator anyway.